features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml

```
@@ -123,7 +123,8 @@ repos:
 
 - name: cuvs
   path: cuvs
-  git: {<<: *git_defaults, repo: cuvs}
+  # TODO(jameslamb): revert this before merging
+  git: {<<: *git_defaults, repo: cuvs, upstream: jameslamb, tag: libcuvs-wheels}
```

```suggestion
  git: {<<: *git_defaults, repo: cuvs}
```

Revert before merging.